src/states/RunCalibrationScript.cpp

```diff
@@ -73,6 +73,15 @@ void RunCalibrationScript::start(mc_control::fsm::Controller & ctl_)
     }
     completed_ = true;
   });
+  // Lower thread priority so that it has a lesser priority than the real time
+  // thread
+  auto th_handle = th_.native_handle();
+  int policy = 0;
+  sched_param param{};
+  pthread_getschedparam(th_handle, &policy, &param);
+  mc_rtc::log::info("[{}] Calibration thread priority: {}", name(), param.sched_priority);
+  param.sched_priority = 80;
+  pthread_setschedparam(th_handle, SCHED_RR, &param);
```

Comment:
This call might fail if the user doesn't have permissions to set such high priority (but in that case it's likely not running in a real-time system). In any case, we shouldn't ignore the call result and display a message if appropriate